### PR TITLE
add: mongodb setup

### LINKED ISSUE #30: Mongo_db url steps
### Preflight Checklist

- [X] I have read the Contributions.md for this project.
- [X] I agree to follow the Code of Conduct that this project adheres to.
- [X] I have searched the feature for a feature request that matches the one I want to file, without success.

### Problem Description

There is no configuration steps of mongo_db url in sample.env file

### Proposed Solution

Create doc contains the configuration steps for mongo_db url

### Alternatives Considered

No

### Additional Information

mention link to this file in the setup section of README.md

### PR DESCRIPTION
**Pull request by amir-ammar**
# Description:

Add mongodb setup

Fixes #30 

## Type of change:

-   [x] This change requires a documentation update

# Checklist:

-   [x] My code follows the style guidelines of this project.
-   [x] I have performed a self-review of my own code.
-   [x] I have commented my code, particularly in hard-to-understand areas.
-   [x] I have made corresponding changes to the documentation.
-   [x] My changes generate no new warnings.
-   [x] I have added tests that prove my fix is effective or that my feature works.
-   [x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

File: docs/MONGO_DB_SETUP.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Mongo_db setup
+1. first thing you need to go to this url [mongo_db login](https://account.mongodb.com/account/login) directly.
```
Comment (Shubham-Kumar-2000) [inline docs/MONGO_DB_SETUP.md]: Points should start with caps ( Capital letters )

Also there are some grammatical errors please solve them. 
@amir-ammar

**Review by Shubham-Kumar-2000 — CHANGES_REQUESTED** [review]

File: docs/MONGO_DB_SETUP.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+
+# Mongo_db setup
+
+1. First thing you need to go to this url [mongo_db login](https://account.mongodb.com/account/login).
+
+  
+
+2. After login (or register incase you donot have account) there will be a page in top of it **Deploy a cloud database** with 3 options (Serverless - Dedicated - Shared) you will choose **Shared** option
```
Comment (Shubham-Kumar-2000) [inline docs/MONGO_DB_SETUP.md]: Its either `do not` or its `don't`.

Please run a spell check

Comment (Shubham-Kumar-2000) [reply docs/MONGO_DB_SETUP.md]: @amir-ammar

**Review by Shubham-Kumar-2000 — CHANGES_REQUESTED** [review]

File: docs/MONGO_DB_SETUP.md (hunk 2 above)
Comment (amir-ammar) [reply docs/MONGO_DB_SETUP.md]: done

**Review by Shubham-Kumar-2000 — APPROVED** [review]

**Pull request merged**